Add slider-based controller for input ports

mntan3 opened the pull request:
# Description
Pass in an arbitrary port for a Drake system and it produces tk sliders to control them. Useful for testing simulation models quickly
![2020-03-23-1584990683_screenshot_229x165](https://user-images.githubusercontent.com/13571695/77354104-96828400-6d18-11ea-8d08-86bb288cdcd5.jpg)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/robotlocomotion/drake/12925)
<!-- Reviewable:end -->

jwnimmer-tri commented:
@mpetersen94 would you me like to sign you on as feature reviewer?

mpetersen94 commented:
@jwnimmer-tri Feel free to add me as a feature reviewer.  I may also ping Russ for review because this is related to lab work.

mntan3 commented:
I addressed all of the comments except for adding `":framework_py"` as a dependency. It seems to build and test fine locally without it. `simple_ui.py` also doesn't have that as a dependency despite also using things from framework.
